includes/polling/applications/unbound.inc.php

```diff
@@ -23,22 +23,26 @@
 $rrd_name =  array('app', $name,'queries',$app_id);
 $rrd_def = RrdDefinition::make()
     ->addDataset('type0', 'DERIVE', 0, 125000000000)
-    ->addDataset('A', 'DERIVE', 0, 125000000000)
-    ->addDataset('NS', 'DERIVE', 0, 125000000000)
```

Comment:
changing all the DNS descriptors to lower case seems not right to me.
They are uppercase everywhere

Reply:
Well, I can't really comment on that, I'm no coder I just changed the file so it works as outlined in the thread on community forums and pasted it here.